### ISSUE TITLE
provide UNC path support from address bar

### ISSUE THREAD

**Issue by 69RandomDude69** [files-community/Files#14172]
### What feature or improvement do you think would benefit Files?

It would be great if UNC paths could be supported as at this point in time, one is forced to operate the standard file explorer alongside files. This is both extremely inconvenient as well as annoying, especially for IT fold who are constantly having to access UNC paths

### Requirements

It would be great if UNC paths could be supported as at this point in time, one is forced to operate the standard file explorer alongside files. This is both extremely inconvenient as well as annoying, especially for IT fold who are constantly having to access UNC paths

### Files Version

3.0.15.0

### Windows Version

10.0.19045.3693

**Comment by yaira2** [comment]
Thank you for the feedback, UNC paths should already be supported but perhaps @Josh65-2201 knows more.

**Comment by Josh65-2201** [comment]
This is working for me. 
- What issue are you having when entering them?
- What's the path your entering into the path?

**Comment by 69RandomDude69** [comment]
Thanks for gettgin back to me!

simple UNC paths. ie. \\hostname\sharename or \\IP address\sharename. In both cases the app just goes into 'not responding' and hangs. I did notice however that leaving for 3-5 minutes actually recovers the app, however it just leaves the folder/files frame blank. I thought it may be an elevation issue so I've tried running the app as a local admin only to be promoted about a limitation of sorts. Same effect.

New development - it seems that some hosts display fine, whereas others do not. The one consistent service that fails across all are printer shares. ie. any host with any printers shared will either not display any shares or only share, but in all cases, not printer shares.

On attempt to close the app whlst 'searching' for the UNC a single crash event is generated as follows:


- System 

  - Provider 

   [ Name]  Windows Error Reporting 
 
  - EventID 1001 

   [ Qualifiers]  0 
 
   Version 0 
 
   Level 4 
 
   Task 0 
 
   Opcode 0 
 
   Keywords 0x80000000000000 
 
  - TimeCreated 

   [ SystemTime]  2023-12-05T23:23:05.2448918Z 
 
   EventRecordID 100582 
 
   Correlation 
 
  - Execution 

   [ ProcessID]  0 
   [ ThreadID]  0 
 
   Channel Application 
 
   Computer 'Removed for privacy' 
 
   Security 
 

- EventData 

    
   0 
   AppHangTransient 
   Not available 
   0 
   Files.exe 
   1.0.0.0 
   6531c3a8 
   unknown 
   unknown 
   unknown 
   unknown 
    
    
    
   \\?\C:\ProgramData\Microsoft\Windows\WER\Temp\WER37C1.tmp.WERInternalMetadata.xml \\?\C:\ProgramData\Microsoft\Windows\WER\Temp\WER383F.tmp.xml \\?\C:\ProgramData\Microsoft\Windows\WER\Temp\WER384D.tmp.csv \\?\C:\ProgramData\Microsoft\Windows\WER\Temp\WER38EB.tmp.txt 
    
    
   0 
   0428ba9f-354e-45f0-8dcd-a98e4a86b66b 
   2049 
    
   0 

I 

the above files are deleted as soon as are generated, making trying to grab a copy of them a tad difficult due to timing. 

Hope the above provides for some assistance
